Project.toml

```diff
@@ -0,0 +1,23 @@
+name = "FFTW"
+uuid = "7a1cc6ca-52ef-59f5-83cd-3a7055c09341"
+version = "0.3.0-DEV"
+
+[deps]
+AbstractFFTs = "621f4979-c628-5d54-868e-fcf4e3e8185c"
+BinaryProvider = "b99e7846-7c00-51b0-8f62-c81ae34c0232"
+Conda = "8f4d0f93-b110-5947-807f-2305c1781a2d"
+Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+
+[compat]
+AbstractFFTs = "≥ 0.3.0"
+BinaryProvider = "≥ 0.3.0"
+julia = "≥ 1.0.0"
```

Comment:
```suggestion
julia = "^1.0"
```

Reply:
I've used the `gen_project.jl` script from `Pkg.jl` to generate this, perhaps it could be improved?  (BTW, what's the difference?)